packages/terra-core-docs/src/terra-dev-site/doc/html-table/About.1.doc.mdx

```diff
@@ -16,8 +17,16 @@ import TableCellPropsTable from 'terra-html-table/lib/TableCell?dev-site-props-t
 
 <Badge />
 
+[![Deprecated](https://badgen.net/badge/status/Deprecated/grey)](https://github.com/cerner/terra-core/tree/main/packages/terra-html-table#readme)
+
 # Terra HTML Table
 
+<Notice variant="deprecation">
+
+`terra-html-table` is no longer supported. If you are using `terra-html-table`, consider replacing it with a supported component.
```

Comment:
Should we suggest they use terra-table? I was thinking of something like:

`terra-html-table` is no longer supported. If you are using `terra-html-table`, consider replacing it with `terra-table` or another supported component.

Reply:
I would agree that including a suggested supported component is a good idea.

Reply:
Thanks for your comments. I updated the notice to refer consumers to Terra-Table.